client/verta/verta/client.py

```diff
@@ -1735,3 +1736,34 @@ def find_datasets(
             datasets = datasets.find(predicates)
 
         return datasets
+
+    def _create_workspace(self, org_id, workspace_name, resource_action_groups):
+        """
+        creates a workspace with a custom role for all users.
+
+        Parameters
+        ----------
+        org_id : str
+            ID of organization in which workspace is created.
+        workspace_name : str
+            name of workspace.
+        resource_action_groups : [_Role.RoleResourceActions]
+            role description with allowed actions for resource types.
```

Comment:
I think this would be more accurate, based on how `custom_role_id` is used.

```suggestion
            Resource actions for non-admins in this workspace.
```